spec/locales-currencies-tz.html

```diff
@@ -52,6 +52,10 @@ <h1>IsStructurallyValidLanguageTag ( _locale_ )</h1>
       <p>
         The abstract operation returns true if _locale_ can be generated from the EBNF grammar in section 3.2 of the Unicode Technical Standard 35, starting with `unicode_locale_id`, and does not contain duplicate variant or singleton subtags (other than as a private use subtag). It returns false otherwise. Terminal value characters in the grammar are interpreted as the Unicode equivalents of the ASCII octet values given.
       </p>
+
+      <emu-note>
+        This function <em>specifically</em> accepts a "Unicode BCP 47 locale identifier", i.e. the backwards compatible syntax from "Unicode CLDR locale identifier" is not accepted. The difference between the two syntaxes is specified in <a href="https://unicode.org/reports/tr35/#BCP_47_Conformance">Unicode Technical Standard 35 section 3.3</a>.
```

Comment:
I suppose this gets the job done, sort of.  But -- it seems like the crux of the confusion is that the bullet points give one explanation for when the function returns *true*; then the paragraph just above this gives a second, and inconsistent, explanation for when the function returns *true*.  The bullets describe a narrower syntax; the paragraph describes a broader syntax.  Why do we have (arguably) two explanations?

Or, stepping back from those nitty-gritty details: if the bullets already say this only recognizes a "Unicode BCP 47 locale identifier", adding a note that has to _repeat_ that claim indicates something has already gone awry.

Or to put it the way I previously noted, https://github.com/tc39/ecma402/pull/429 would address this concern by listing the requirements of the algorithm, then leaving the _implications_ of those requirements to a note that neither reiterates what those requirements say, nor (even implicitly) contradicts them.